frontend/components/software/edit/information/ValidateConceptDoi.tsx

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: 2022 Dusan Mijatovic (dv4all)
+// SPDX-FileCopyrightText: 2022 dv4all
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import {useState} from 'react'
+import Button from '@mui/material/Button'
+import CircularProgress from '@mui/material/CircularProgress'
+import FactCheckIcon from '@mui/icons-material/FactCheck'
+
+import {softwareInformation as config} from '../editSoftwareConfig'
+import useSnackbar from '~/components/snackbar/useSnackbar'
+import {getSoftwareVersionInfoForDoi} from '~/utils/getDataCite'
+
+export default function ValidateConceptDoi({doi}: { doi: string }) {
+  const {showErrorMessage,showSuccessMessage, showInfoMessage} = useSnackbar()
+  const [loading,setLoading]=useState(false)
+
+  async function validateDoi() {
+    setLoading(true)
+    const info = await getSoftwareVersionInfoForDoi(doi)
+    if (info) {
+      if (info.versionOfCount === 0) {
+        showSuccessMessage(`Valid concept DOI: ${doi}`)
+      } else if (info.versionOfCount === 1) {
+        const conceptDoi = info.versionOf.nodes[0].doi
+        showErrorMessage(`This is version DOI. The concept DOI is ${conceptDoi}`)
+      }
+    } else {
+      showErrorMessage(`Failed to retereive version info for DOI: ${doi}. This is high likely not concept DOI.`)
```

Comment:
I suggest to rephrase a little bit:

```suggestion
      showErrorMessage(`Failed to retrieve version info for DOI: ${doi}. It looks like this is not a Concept DOI.`)
```